1449 improve examples (#1661)
open-contracting/standard: docs/schema/merging.md

A **versioned value** thus describes a field's value in a specific release.

For example, in worked example 1 in [Updates and deletions](../guidance/build/merging), the estimated value of the procurement was 13,000 GHS (in the first release `tender/value/amount` was `13000`). Following the steps above, the versioned value is:

```json
{
  "releaseID": "ocds-uhveoc-144560-1601380494627-tender",
  "releaseDate": "2020-10-21T13:02:26Z",
  "releaseTag": [
    "tender"
  ],
  "value": 13000
}

> ```

Reviewer: Rather than linking to the worked example and including only the versioned value on this page, I think the purpose of this example (illustrating the relationship between a release and a versioned release) would be better served by adding a `jsoninclude` directive showing the relevant fields in the release (`id`, `date`, `tag` and `tender.value.amount`).

Reviewer: Noting that this comment is still to be addressed.

Author: ah, I thought I'd done that but if you're saying I've not then I'm not sure how to

Reviewer: I think that the purpose of the example is to illustrate the relationship between a release and a versioned release. Presently, the only JSON example on the page is the versioned release so if users want to see what the associated release looks like they need to follow the link to the updates and deletions worked example. My suggestion is to replace the link to the updates and deletions worked example with a `jsoninclude` directive showing the relevant fields in the release so that readers can see both the release and versioned release on the same page. Does that make sense?

Author: ooh, right I see, sorry it makes sense now and I agree it makes the point much clearer in the docs

Author: Just noting that I'll push this change and the other similar one once there's agreement about changing this to be fictional (see comment in https://github.com/open-contracting/standard/pull/1661#discussion_r1421936525

Author: should be fixed now

Reviewer: Resolving as I've updated this example.

In a **versioned release**, with a few exceptions, a field's value is replaced with an array of versioned values, which should be in chronological order by `releaseDate`.

For example, in the worked example referenced above, the number of tenderers was 0 in the release published October 21, 2020 and then 2 in the release published November 5, 2020. In a versioned release, this is serialized as below:

```{jsoninclude} ../examples/merging/updates/ghana_versioned.json
:jsonpointer: /records/0/versionedRelease/tender
:expand: numberOfTenderers,releaseTag
:title: Versioned_values
```

```{jsoninclude} ../examples/merging/updates/ghana_versioned.json
:jsonpointer:
:expand: records, versionedRelease

> :title: Versioned_release

Reviewer: As above, I would include `jsoninclude` directives showing the relevant fields in the releases. Similar to my other comments, I would significantly reduce the number of fields in the examples files to help readers focus on the relevant parts.

Reviewer: Noting that this comment is still to be addressed.

Author: as above, I thought I'd done this but if not then I'm not sure how to

Author: this should be right now

Reviewer: Resolving as I've updated this example.

A **versioned value** thus describes a field's value in a specific release.

> For example, a tender release sets the value of `tender.value.amount`.

Reviewer 2: ```suggestion
For example, a tender release sets the `tender.value.amount`field:
```

```{jsoninclude} ../examples/amendments/tender.json
:jsonpointer: /records/0/releases/0
:expand: tag, tender, value
```

> Following the steps above, the versioned value of `tender.value.amount` is:

Reviewer 2: ```suggestion
Following the steps above, the versioned value of the `tender.value.amount` field is:
```

```{jsoninclude} ../examples/amendments/tender.json
:jsonpointer: /records/0/versionedRelease/tender/value/amount/0
:expand: releaseTag
```

In a **versioned release**, with a few exceptions, a field's value is replaced with an array of versioned values, which should be in chronological order by `releaseDate`.

> Following on from the example above, a later release updates the value of `tender.value.amount`.

Reviewer 2: ```suggestion
Following on from the example above, a later release updates the value of the `tender.value.amount` field:
```

```{jsoninclude} ../examples/amendments/tender.json
:jsonpointer: /records/0/releases/2
:expand: tag, tender, value
```

> In the versioned release, `tender.value.amount` is replaced with an array of versioned values:

Reviewer 2: ```suggestion
In the versioned release, the `tender.value.amount` field is an array, that now contains another versioned value:
```
